include/seastar/core/sharded.hh

```diff
@@ -952,6 +957,7 @@ private:
     PtrType _value;
     unsigned _cpu;
 private:
+    constexpr static bool has_use_count = HasUseCount<PtrType>;
```

Comment:
This is no longer needed in this patch?

Reply:
dropped in [affaf8b](https://github.com/scylladb/seastar/commit/affaf8b6129d4784036aae4878cb949c3ab29a14)